File: src/main/java/edu/spectrum/SpecHelper.java
package edu.spectrum;

import javax.swing.*;

import edu.spectrum.gui.SpecHelperFrame;
import edu.spectrum.logging.Messager;

public class SpecHelper {
    /**
     * @param args
     *            the command line arguments
     */
    public static void main(String args[]) {
        try {
            for (javax.swing.UIManager.LookAndFeelInfo info : javax.swing.UIManager
                    .getInstalledLookAndFeels())
                if ("Windows".equals(info.getName())) {
                    UIManager.setLookAndFeel(info.getClassName());
                    break;
                }
        } catch (ClassNotFoundException ex) {
            Messager.publish("SYNTHelper - main", ex);
        } catch (InstantiationException ex) {
            Messager.publish("SYNTHelper - main", ex);
        } catch (IllegalAccessException ex) {
            Messager.publish("SYNTHelper - main", ex);
        } catch (UnsupportedLookAndFeelException ex) {
            Messager.publish("SYNTHelper - main", ex);
        }
        // </editor-fold>

		/* Create and display the form */
        java.awt.EventQueue.invokeLater(new Runnable() {
            @Override
            public void run() {
                SpecHelperFrame app = new SpecHelperFrame();
                app.setVisible(true);
                app.setExtendedState(app.getExtendedState()
                        | JFrame.MAXIMIZED_BOTH);
            }
        });
    }
}

File: src/main/scala/edu/spectrum/SpecHelper.scala
package edu.spectrum

import java.awt.Frame
import javax.swing._

import edu.spectrum.gui.SpecHelperFrame
import edu.spectrum.logging.Messager

object SpecHelper extends App {
    try {
      UIManager
        .getInstalledLookAndFeels
        .find(_.getName == "Windows")
        .foreach(info => UIManager.setLookAndFeel(info.getClassName))
    }
    catch {
      case ex: ClassNotFoundException =>
        Messager.publish("SYNTHelper - main", ex)
      case ex: InstantiationException =>
        Messager.publish("SYNTHelper - main", ex)
      case ex: IllegalAccessException =>
        Messager.publish("SYNTHelper - main", ex)
      case ex: UnsupportedLookAndFeelException =>
        Messager.publish("SYNTHelper - main", ex)
    }

    /* Create and display the form */
    java.awt.EventQueue.invokeLater(
      new Runnable() {
        def run() {
          val app: SpecHelperFrame = new SpecHelperFrame
          app.setVisible(true)
          app.setExtendedState(app.getExtendedState | Frame.MAXIMIZED_BOTH)
        }
      })
}